int16 drone_id
int32 traj_id
time start_time
float32 des_clearance

uint8 order
float32[3] start_p
float32[3] start_v
float32[3] start_a
float32[3] end_p
float32[3] end_v
float32[3] end_a
float32[] inner_x
float32[] inner_y
float32[] inner_z
float32[] duration